graphql/resolve/mutation.go

```diff
@@ -222,7 +222,7 @@ func (mr *dgraphResolver) rewriteAndExecute(
 	commit := false
 
 	defer func() {
-		if !commit && mutResp != nil && mutResp.Txn != nil {
+		if !commit && mutResp != nil && mutResp.Txn != nil && mutResp.Txn.StartTs > 0 {
```

Comment:
Let's remove this StartTs > 0 check. StartTs always be greater than zero.

Reply:
Done